source/form.html

```diff
@@ -0,0 +1,126 @@
+  <form class="page-main__form form" action="https://echo.htmlacademy.ru" method="post">
+    <fieldset class="form__fieldset cat-info-fieldset">
+      <ul class="form__list cat-info-list">
+        <li class="cat-info-list__item">
+          <label class="form__cat-info-label" for="cat-name">Имя:*</label>
+          <input class="form__cat-info-input" id="cat-name" name="cat-name" placeholder="Барсик" type="text" required>
+        </li>
+        <li class="cat-info-list__item">
+          <label class="form__cat-info-label" for="cat-weight">Вес (кг):*</label>
+          <input class="form__cat-info-input" id="cat-weight" name="cat-weight" placeholder="7" type="number" required>
+        </li>
+        <li class="cat-info-list__item">
+          <label class="form__cat-info-label" for="cat-age">Возраст (лет):</label>
+          <input class="form__cat-info-input" id="cat-age" name="cat-age" placeholder="7" type="number">
+        </li>
+      </ul>
+    </fieldset>
+    <fieldset class="form__fieldset">
+      <ul class="form__list cat-mass-list">
+        <li class="cat-mass-list__item">
+          <input class="form__cat-mass-input" id="weight-loss" name="about-weight" type="radio" value="weight-loss" checked>
+          <label class="form__cat-mass-label" for="weight-loss">Похудение</label>
+        </li>
+        <li class="cat-mass-list__item">
+          <input class="form__cat-mass-input" id="weight-gain" name="about-weight" type="radio" value="weight-gain">
+          <label class="form__cat-mass-label" for="weight-gain">Набор массы</label>
+        </li>
+        <li class="cat-mass-list__item">
+          <input class="form__cat-mass-input" id="idk-help-me" name="about-weight" type="radio" value="i-dont-know">
+          <label class="form__cat-mass-label" for="idk-help-me">Не знаю (Нужен ваш совет)</label>
+        </li>
+      </ul>
+    </fieldset>
+    <fieldset class="form__fieldset">
+      <h2 class="form__headline thematic-headline">Контактные данные (владельца кота)</h2>
+      <ul class="form__list owner-info-list">
+        <li class="owner-info-list__item">
+          <label class="form__owner-info-label" for="email">E-mail:*</label>
+          <input class="form__owner-info-input form__owner-info-input--letter" id="email" name="email" type="email" required>
+        </li>
+        <li class="owner-info-list__item">
+          <label class="form__owner-info-label" for="telephone">Телефон:*</label>
+          <input class="form__owner-info-input form__owner-info-input--telephone" id="telephone" name="telephone" type="tel" required>
+        </li>
+      </ul>
+    </fieldset>
+    <fieldset class="form__fieldset">
+      <h2 class="form__headline thematic-headline">Комментарий</h2>
+      <textarea class="form__text-about-cat-textarea textarea" id="about-cat" name="about-cat" placeholder="Расскажите обо всех повадках кота"></textarea>
```

Comment:
```suggestion
      <textarea class="form__textarea" id="about-cat" name="about-cat" placeholder="Расскажите обо всех повадках кота"></textarea>
```
она не самостоятельный блок